<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sonerit</class>
 <widget class="QMainWindow" name="sonerit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>1043</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sonerit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1260</width>
      <height>1010</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>860</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continue_button">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>860</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CONTINUE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>920</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2 Player</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit_player1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>880</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit_player2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>940</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>860</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Player 1 Name</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>920</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Player 2 Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player1_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_player2_name">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_points">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>POINTS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TIME LEFT:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_health1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_health2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_points">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_time">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_gameover">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>841</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_llt">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_llt">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SUDDEN DEATH TIME:</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>pushButton</zorder>
   <zorder>continue_button</zorder>
   <zorder>checkBox</zorder>
   <zorder>textEdit_player1</zorder>
   <zorder>textEdit_player2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_player1_name</zorder>
   <zorder>label_player2_name</zorder>
   <zorder>label_points</zorder>
   <zorder>label_2</zorder>
   <zorder>label_time</zorder>
   <zorder>lcd_health1</zorder>
   <zorder>lcd_health2</zorder>
   <zorder>lcd_points</zorder>
   <zorder>lcd_time</zorder>
   <zorder>label_gameover</zorder>
   <zorder>lcd_llt</zorder>
   <zorder>label_llt</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
